src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenIfOperator.vb

```diff
@@ -15,6 +15,567 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
     Public Class CodeGenIfOperator
         Inherits BasicTestBase
 
+        <Fact, WorkItem(61483, "https://github.com/dotnet/roslyn/issues/61483")>
```

Comment:
Consider also testing with IntPtr in VB (optimization should not kick in, unlike C# in some cases)